pkg/ccl/changefeedccl/event_processing.go

```diff
@@ -495,6 +539,32 @@ func (c *parallelEventConsumer) getBucketForEvent(ev kvevent.Event) int64 {
 	return int64(c.hasher.Sum32()) % c.numWorkers
 }
 
+func (c *parallelEventConsumer) dispatcher(ctx context.Context, sink EventSink) error {
```

Comment:
nit: (well, maybe not a nit, but a request) I got confused initially thinking that dispatcher dispatches the work to the workers... where as in reality, it is the workers that emit to this dispatcher which then writes to the underlying sink -- thus ensuring sink usage is thread safe.  I think it would be clearer if this function was renamed emitRowWorker -- and, while at it, a comment added along the lines of what I typed above.

Reply:
Renamed to `emitRowWorker` and I added a comment